change/@ni-eslint-config-angular-f944c513-51ac-41b7-adc9-be562b54896b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "major",
```

Comment:
I'm marking this as a "major" change since this PR drops support for a version of Angular that was previously supported. I doubt anyone was actually using this package within an Angular 15 project, but that change type still seemed to make the most sense to me. If anyone disagrees, I'd be happy to discuss.

Reply:
Was kinda hoping to treat it as a patch i.e. the version upgrade had a bug that is being reverted so users don't have issues, which is pretty much true

Reply:
If we're happy viewing it with that perspective, that's fine with me. I can change this to "patch".

Reply:
I'll give Jesse final say since he's been more involved with teams CI issues

Reply:
@mollykreis actually yea, lets go with that approach, think potential fallout is minimal and workaround will be available quickly (switch to the quickly upcoming major release)